docker/readme.rst

```diff
@@ -66,7 +66,7 @@ everest playground
 
 If you would like to get a pre-configured development setup using
 *vscode* for the *everest-cpp* framework, you need to start up the mqtt
-server with::
+server with:
```

Comment:
I think this double-colon is intentional: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks

Reply:
Thank you, I didn't find that rst documentation, and failed to recognize the formatting change.